docs/components/page-higher-order/DocumentationElements.tsx

```diff
@@ -21,6 +21,10 @@ export default function DocumentationElements(props: DocumentationElementsProps)
     headings: props.headings,
   });
 
+  if (!props.meta) {
+    return props.children;
+  }
+
```

Comment:
I guess it's a workaround which hides the root error, in general, all pages should have the `meta` prop set.

Can you explain why those lines ware needed?

Reply:
I added MDX docs that are not full documents, and are instead fragments used inside of UI components. These fragments don't need meta data or any of the providers that full docs do, so I added this exception.